<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../2C8F-037C/Mis Projectos/Almacen Peaje/BDPeatge Python/Imagenes/Logo.bmp</normaloff>../../../../2C8F-037C/Mis Projectos/Almacen Peaje/BDPeatge Python/Imagenes/Logo.bmp</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>781</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="L_Revisado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>REVISADO:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_Revisado">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>151</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="L_CAlmacen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COD.ALMACEN:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_CAlmacen">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>201</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="L_PComanda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PUNTO DE COMANDA:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_PComanda">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>301</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="L_CFabrica">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COD.FABRICA:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_CFabrica">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>251</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="GB_Stock">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>349</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="title">
     <string>STOCK</string>
    </property>
    <widget class="QLabel" name="L_Almacen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ALMACEN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_Calderas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CALDERAS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_Total">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TOTAL:</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_Washinton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>WASHINTON:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_Observaciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>OBSERVACIONES</string>
    </property>
    <widget class="QPlainTextEdit" name="E_Observaciones">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>151</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_Codigo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>CODIGO</string>
    </property>
    <widget class="QTextEdit" name="E_Codigo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="E_Total">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_Calderas">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_Washinton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="E_Almacen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>720</y>
     <width>761</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="B_Tabla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TABLA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Salir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>761</width>
     <height>691</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="L_Grupo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GRUPO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Descripcion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>630</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DESCRIPCIÃ“N:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_Descripcion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>501</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_Grupo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="minimumContentsLength">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Anterior">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Siguiente">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>31</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Foto">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>20</y>
      <width>650</width>
      <height>550</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Imagen1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGEN 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Imagen2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGEN 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Imagen3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>580</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGEN 3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
